shared/src/metabase/mbql/schema.cljc

```diff
@@ -713,13 +713,15 @@
   amount   NumericExpressionArg
   unit     ArithmeticDateTimeUnit)
 
+(defclause ^{:requires-features #{:date-arithmetics}} now)
```

Comment:
I'm not sure that `now` should require date-arithmetic features. Seems like almost all dbs support a `now()` function right? The implementation to add and subtract datetimes seems independent of `now`? Or do they go hand-in-hand?

Reply:
Good question. All you said is true, but the reason now is bundled with `:date-arithmetics` is that in order for 3rd party drivers to opt in to implementing it, it has to be bundled with _some_ feature flag, and date-arithmetics probably makes the most sense. 

see [this discussion](https://metaboat.slack.com/archives/C03SHMZ266R/p1668622166578089)